assets/src/components/AssignmentTable.js

```diff
@@ -92,16 +104,27 @@ function AssignmentTable (props) {
     handleInputBlur
   } = props
 
-  const [popoverEl, setPopoverEl] = useState({ popoverId: null, anchorEl: null })
+  const assignmentStatus = {
+    GRADED: 'Graded',
+    SUBMITTED: 'Not Yet Graded',
+    UNSUBMITTED: 'Unsubmitted'
+  }
```

Comment:
This Object can be separated out and can be put in a `utils/assignments.js` and be reused in `AssignmentTable.js` and in the `AssignmentPlanningV2.js`